src/vm/eeconfig.cpp

```diff
@@ -789,10 +792,18 @@ HRESULT EEConfig::sync()
         g_fEnableARM = TRUE;
     }
 
-    int forceGCconcurrent = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_gcConcurrent);
-    if ((forceGCconcurrent > 0) || (forceGCconcurrent == -1 && g_IGCconcurrent))
+    bool gcConcurrentWasForced = false;
+#ifdef FEATURE_CORECLR
```

Comment:
If we revert the change in clrconfigvalues.h, the startup flag for concurrent GC will not be enabled unless "System.GC.Concurrent" is explicitly set in the JSON configuration. We can change the code (in unixinterface.cpp) that sets the flag so that it uses the boolean argument fallback instead of the COMPlus one and always returns true if the value isn't explicitly set, but then you won't be able to use the COMPlus mechanism to configure the concurrent GC startup flag.

If we keep the change in clrconfigvalues.h, the existing code that checks for -1 doesn't make sense anymore for CoreCLR.